src/channel.cpp

```diff
@@ -28,6 +28,7 @@
 // CChannel implementation *****************************************************
 CChannel::CChannel ( const bool bNIsServer ) :
     vecdGains              ( MAX_NUM_CHANNELS, 1.0 ),
+	vecdPannings		   ( MAX_NUM_CHANNELS, 0.5),
```

Comment:
You seem to work with tabs. In the Jamulus source code, I replace tabs with spaces everywhere.